### PR TITLE
Backtick table name in generic orphan check

### LINKED ISSUE #20018: gitea doctor: pq: syntax error at or near "." whilst counting Orphaned Stopwatches without existing User
### Description

Full doctor output:
```sh
git@localhost:/var/lib/gitea$ gitea doctor --all --config /etc/gitea/app.ini -w . -C ./custom --fix
[1] Check paths and basic configuration
 - [I] Configuration File Path:    "/etc/gitea/app.ini"
 - [I] Repository Root Path:       "/var/lib/gitea/data/gitea-repositories"
 - [I] Data Root Path:             "data"
 - [I] Custom File Root Path:      "custom"
 - [I] Work directory:             "."
 - [I] Log Root Path:              "/var/lib/gitea/log"
OK
[2] Check if there is garbage storage files
OK
[3] Check Database Version
OK
[4] Check consistency of database
 - [C] Error: pq: syntax error at or near "." whilst counting Orphaned Stopwatches without existing User
ERROR
[5] Check if user with wrong type exist
OK
[6] Check if OpenSSH authorized_keys file is up-to-date
OK
[7] Check if SCRIPT_TYPE is available
 - [I] ScriptType bash is on the current PATH at /usr/bin/bash
OK
[8] Check if hook files are up-to-date and executable
OK
[9] Recalculate Stars number for all user
 - [I] Updated User Stars numbers.
OK
[10] Check old archives
 - [I] 0 / 0 old archives in repository deleted
OK
[11] Check that all git repositories have receive.advertisePushOptions set to true
 - [I] Enabled push options for 4 repositories.
OK
[12] Check for incorrectly dumped repo_units (See #16961)
 - [I] Fixed 0 broken repo_units
OK
[13] Recalculate merge bases
 - [I] 0 PR mergebases updated of 16 PRs total in 4 repos
OK
[14] Check git-daemon-export-ok files
 - [I] Updated git-daemon-export-ok files for 0 of 4 repositories.
OK
[15] Check if users has an valid email address
 - [I] All users have a valid e-mail.
OK
```

### Gitea Version

1.17.0+dev-751-g433443ffa built with GNU Make 4.3, go1.18.1 : bindata

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

Locally compiled binary

### Database

PostgreSQL

### PR DESCRIPTION
**Pull request by Gusted**
- Resolves #20018

### CODE REVIEW

File: modules/doctor/dbconsistency.go [hunk 1]
```diff
@@ -201,10 +201,10 @@ func checkDBConsistency(ctx context.Context, logger log.Logger, autofix bool) er
 			"oauth2_authorization_code", "oauth2_grant", "oauth2_authorization_code.grant_id=oauth2_grant.id"),
 		// find stopwatches without existing user
 		genericOrphanCheck("Orphaned Stopwatches without existing User",
-			"stopwatch", "user", "stopwatch.user_id=user.id"),
+			"stopwatch", "user", "stopwatch.user_id=`user`.id"),
 		// find stopwatches without existing issue
 		genericOrphanCheck("Orphaned Stopwatches without existing Issue",
-			"stopwatch", "issue", "stopwatch.issue_id=issue.id"),
+			"stopwatch", "issue", "stopwatch.issue_id=`issue`.id"),
```
Comment (delvh) [inline modules/doctor/dbconsistency.go:207]: Does issue need this as well?

**Review by delvh — APPROVED** [review]

File: modules/doctor/dbconsistency.go (hunk 1 above)
Comment (Gusted) [reply modules/doctor/dbconsistency.go:207]: No harm 🤷🏽(Simple ctrl-f shows multiple references and I have no idea when to use them 😄) I don't have a Postgres setup to test it.

Comment (6543) [reply modules/doctor/dbconsistency.go:207]: it's not consistent before and wont be after ... - I dont mid if this is changed too

**Review by 6543 — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Pull request merged**